llvm/test/CodeGen/SPIRV/inline/type.coop-matrix.ll

```diff
@@ -0,0 +1,28 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
```

Comment:
nit: add a TODO to enable spirv-val once we get the capability in (or we add inline-spirv support to set it from the test file)

Reply:
Done.